File: Baitap1/Complete/Python/7.2.py
def is_even(i):
    """
    Input: i, a positive int
    Returns True if i is even, otherwise False
    """
    remainder = i % 2
    return remainder == 0

print("All numbers between 0 and 20: even or not")
for i in range(20):
    if is_even(i):
        print(i, "even")
    else:
        print(i, "odd")

File: Baitap1/Complete/Java/7.2.java
public class Main {
    private static boolean isEven(int x) {
        int remainder = x % 2;
        return remainder == 0;
    }
    public static void main(String[] args) {
        System.out.println("All numbers between 0 and 20: even or not");
        for(int i = 0; i < 20; i ++) {
            if (isEven(i))
                System.out.println(i + " even");
            else
                System.out.println(i + " odd");
        }
    }
}